website/server/controllers/api-v3/chat.js

```diff
@@ -229,6 +229,8 @@ api.postChat = {
     if (group.type === 'party') {
       user.party.lastMessageSeen = newChatMessage.id;
       toSave.push(user.save());
+
+      sendChatPushNotifications(user, group, newChatMessage, res.t);
```

Comment:
Can this be moved to inside the `group.sendChat ` function code? Since the other notifications code is there already